src/asciidoc/docs/guides-general/enable-file-uploads.adoc

```diff
@@ -0,0 +1,146 @@
+= Enabling file uploads
+:page-slug: enable-file-uploads
+:page-description: How to enable file uploads to the CDN through your Koji app's frontend.
+:includespath: ../_includes
+
+If users need to upload files through your app, you can use several built-in controls, depending on the expected type of the uploaded file.
+
+The @withkoji/core package provides two Frontend classes for file uploads: <<core-frontend-ui-capture#,UI / Capture>> and <<core-frontend-ui-upload#,UI / Upload>>.
+
+This topic helps you choose the right controls for specific file types and scenarios.
+
+* <<_only_one_media_type,If your app accepts only one media type (audio, image, or video).>>
+** <<_only_audio_files,Only audio files.>>
+** <<_only_images,Only images.>>
+** <<_only_videos,Only videos.>>
+* <<_two_or_more_media_types,If your app accepts two or more media types (audio, image, or video).>>
+* <<_any_file_as_is,If your app accepts any file as is, typically for downloading (not rendered).>>
+* <<_generated_content,If your app generates new content (audio, image, or video) that needs to be saved.>>
+
+
+== Only one media type
+=== Only audio files
+
+If your app accepts only audio files, use `Koji.ui.capture.audio`.
```

Comment:
I think it would make more sense to use `upload control` or just `upload` instead of `app` here and in the other places this construction appears, as an app might include multiple upload scenarios of different types.

Reply:
Fixed. 

Kept 'app' in the last section which says "If your app generates new content...."